blivet/flags.py

```diff
@@ -64,6 +64,10 @@ def __init__(self):
 
         self.multipath_friendly_names = True
 
+        # set to None since blivet doesn't need to account for any
+        # additional blacklisted devices by default
+        self.blacklisted_devs = None
```

Comment:
An empty list would be a better default value here. And it would also allow you to remove the pylint-disable thing below.